core/src/main/scala/com/databricks/labs/remorph/generators/sql/ExpressionGenerator.scala

```diff
@@ -371,11 +371,21 @@ class ExpressionGenerator extends BaseSQLGenerator[ir.Expression] with Transform
 
   private def in(inExpr: ir.In): SQL = {
     val values = commas(inExpr.other)
-    code"${expression(inExpr.left)} IN ($values)"
+    val enclosed = values.map { sql =>
+      if (sql.charAt(0) == '(' && sql.charAt(sql.length - 1) == ')') {
```

Comment:
nitpicking, as it is unlikely to happen, but `str.charAt(0) == '('` would fail on the empty string, as opposed to `str.startsWith("(")`

Reply:
can be done in another PR fixing #1252